app/controllers/users_controller.rb

```diff
@@ -251,6 +249,8 @@ def account_report
   end
 
   def reset_password
+    @user = User.find(params[:id])
+    authorize @user, :update_edit_or_destroy?
```

Comment:
This would be better to move into the UserPolicy since only managers should be able to destroy Users. So even though this line does the right thing it is confusing.